apps/mymedicare_cb/models.py

```diff
@@ -61,71 +61,92 @@ def get_and_update_user(
 
     log_dict = {
         "type": "mymedicare_cb:get_and_update_user",
+        "subject": subject,
```

Comment:
Added this to the logging. It is the SLSx subject (unique username/primary id key) for the beneficiary.